app/lib/irs1040_pdf.rb

```diff
@@ -10,7 +10,7 @@ def initialize(submission)
     @tax_return = submission.tax_return
     @intake = submission.intake
     @qualifying_dependents = submission.qualifying_dependents
-    @address = @submission.address
+    @address = @intake.address
```

Comment:
i started writing this reponse about how i explored that and it wouldnt work, but i mistakenly thought that the address was associated with the INTAKE and not the submission, so i thought there was going to be some out-of-sync-ness. But, this does make sense. However, because of the way we do the PDF generation, the vast majority of clients WONT have a verified address when we create the PDF because we do the address verification later... i can change that logic around, but it might get a little messy...